src/main/resources/templates/pollingunits.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <style>
+    table{width: 100%;}
```

Comment:
I strongly recommend externalizing this block

Reply:
Will do. @sfagade